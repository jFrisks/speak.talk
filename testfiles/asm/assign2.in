A function that is called main. It returns an integer by running the following code:
  - an integer that is called x.
  - an integer that is called y and has the value of 10;
  - the variable that is called x = the variable that is called y added to 2;
  - run the function that is called print and uses the arguments of 42;
  - finally return the value of the function f that uses the arguments of the variable that is called y added to x;
.

A function that is called f and uses the arguments of an integer that is called x. It returns an integer by running the following code:
  - run the function that is called print and uses the arguments of the variable that is called x;
  - an integer that is called y and has the value of 2;
  - run the function that is called print and uses the arguments of the variable that is called y;
  - finally return 1;
.