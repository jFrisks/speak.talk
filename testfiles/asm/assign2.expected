42
22
2